internal/olm/operator/bundle/install.go

```diff
@@ -184,13 +185,32 @@ func generateFBCContent(ctx context.Context, f *fbcutil.FBCContext, bundleImage,
 	return content, nil
 }
 
+// below is duplicate code and same added in util.go try to merge both
+func nullLogger() *log.Entry {
+	logger := log.New()
+	logger.SetOutput(ioutil.Discard)
+	return log.NewEntry(logger)
+}
+
```

Comment:
I noticed the comment here mentioning duplicated code. Should we consider putting the implementation of this into a separate utility package? We could potentially add the `containerdregistry` creation to a utility function as well to limit the amount of code duplication.

What do you think @laxmikantbpandhare?

Reply:
yeah, while writing code I mentioned this comment and the forgot hehe. I will add it in `utility.go` and remove duplicated code.

Reply:
Updated.